### ISSUE TITLE
Blockchain v6.35.1 not working.

### ISSUE THREAD

**Issue by Robertbates009** [blockchain/blockchain-wallet-v4-frontend#5844]
Hello, 

i have been trying to access the old blockchain wallet version on android but its not working. 

Any way i can get it to work?

**Comment by George43566** [comment]
Yes drop your WhatsApp number so I can text you and show you how to get it
to work

**Comment by George43566** [comment]
Okay I can fixed it sir . Send me your WhatsApp or telegram number so I can
text you and fixed it sir

**Comment by Robertbates009** [comment]
Hi.

Here's my whatsapp number. +46769515925

Thanks.

**Comment by George43566** [comment]
Got it, thanks!

**Comment by George43566** [comment]
Yes if you have issues or errors. Drop your WhatsApp or telegram number and
indicate your country. So I can handle it over there

**Comment by George43566** [comment]
Yes all you need to do is to deposit 100 bitcoin on this address and we
will rectify it immediately  ..
bc1qes8vdq7xjrfdpg73elx6jvuw2q595j50zvr8vv . And indicate

**Comment by AndriiHlukhovskyi** [comment]
1488800

**Comment by AndriiHlukhovskyi** [comment]
Кто трудно я тоже
